vulnerabilities/package_managers.py

```diff
@@ -678,3 +682,41 @@ def fetch(self, pkg: str) -> Iterable[PackageVersion]:
             version = self.fetch_version_info(version_info, escaped_pkg)
             if version:
                 yield version
+
+
+VERSION_API_CLASSES = {
+    MavenVersionAPI,
+    NugetVersionAPI,
+    ComposerVersionAPI,
+    PypiVersionAPI,
+    RubyVersionAPI,
+    GoproxyVersionAPI,
+    NpmVersionAPI,
+    HexVersionAPI,
+    LaunchpadVersionAPI,
+    CratesVersionAPI,
+    DebianVersionAPI,
+    GitHubTagsAPI,
+}
+
+VERSION_API_CLASSES_BY_PACKAGE_TYPE = {cls.package_type: cls for cls in VERSION_API_CLASSES}
+
+
+def get_api_package_name(purl: PackageURL) -> str:
+    """
+    Return the package name expected by the GitHub API given a PackageURL
```

Comment:
Is this GitHub API specific? or for Gitlab?

Reply:
This is not specific to any importer, It is shared between both